app/src/androidTest/java/org/mozilla/fenix/helpers/TestAssetHelper.kt

```diff
@@ -17,11 +17,8 @@ object TestAssetHelper {
     @Suppress("MagicNumber")
     val waitingTime: Long = TimeUnit.SECONDS.toMillis(15)
     val waitingTimeShort: Long = TimeUnit.SECONDS.toMillis(1)
-    // File name length in most operating systems is limited to 255 bytes,
-    // including file extension.
-    // This string is 251 char long though, as Android Studio won't compile
-    // with a resource file having a name longer than 251 chars.
-    const val downloadFileName = "tAJwqaWjJsXS8AhzSninBMCfIZbHBGgcc001lx5DIdDwIcfEgQ6vE5Gb5VgAled17DFZ2A7ZDOHA0NpQPHXXFHPSD4wzCkRWiaOorNI574zLtv4Hjiz6O6T7onmUTGgUQ2YQoiQFyrCrPv8ZB9KvmtoRFRVIZh8Pg2a1THrm9gpMoLwP44nDNN5o70USUyVXFxPeTerG9OSdpZIwCWGIVM3AZmjFSaiAyZKxJp8G1oZ2md4DBQYy6F1.svg"
+    // A long enough file name to not fit on a single line in the UI.
```

Comment:
This change would definitely help a lot of windows users. But I had an intention to test not only that a long line would fit horizontally (which was a reported issue), but also vertically (which could be a potential issue), hence the usage of 255 file name length (or as close as we can get to that).